.github/workflows/CI.yml

```diff
@@ -99,6 +99,11 @@ jobs:
           npm --version
           npx commitlint --version
           yarn --version
+          npx tsc --version
+      - name: Run typescript compiler
+        # This step is needed because jest will swallow any compiler error
```

Comment:
@webwarrior-ws shouldn't this say "would" instead of "will"?

Reply:
It will swallow a compiler error, if there is any.

Reply:
yes but after this commit it will not; so putting this comment is misleading, change the "will" with "would" please

Reply:
Done.